sdk/cosmos/azure-cosmos/test/test_feed_range.py

```diff
@@ -0,0 +1,117 @@
+# The MIT License (MIT)
+# Copyright (c) Microsoft Corporation. All rights reserved.
+import unittest
+import uuid
+
+import pytest
+
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.partition_key as partition_key
+import test_config
+
+from azure.cosmos._change_feed.feed_range_internal import FeedRangeInternalEpk
+from azure.cosmos._routing.routing_range import Range
+
+@pytest.fixture(scope="class")
+def setup():
+    if (TestFeedRange.masterKey == '[YOUR_KEY_HERE]' or
+            TestFeedRange.host == '[YOUR_ENDPOINT_HERE]'):
+        raise Exception(
+            "You must specify your Azure Cosmos account values for "
+            "'masterKey' and 'host' at the top of this class to run the "
+            "tests.")
+    test_client = cosmos_client.CosmosClient(TestFeedRange.host, test_config.TestConfig.masterKey),
+    created_db = test_client[0].get_database_client(TestFeedRange.TEST_DATABASE_ID)
+    return {
+        "created_db": created_db,
+        "created_collection": created_db.get_container_client(TestFeedRange.TEST_CONTAINER_ID)
+    }
+
+test_subset_ranges = [(Range("", "FF", True, False),
+                       Range("3F", "7F", True, False),
+                       True),
+                      (Range("3F", "7F", True, False),
+                       Range("", "FF", True, False),
+                       False),
+                      (Range("3F", "7F", True, False),
+                       Range("", "5F", True, False),
+                       False),
+                      (Range("3F", "7F", True, True),
+                       Range("3F", "7F", True, True),
+                       True),
+                      (Range("3F", "7F", False, True),
+                       Range("3F", "7F", True, True),
+                       False),
+                      (Range("3F", "7F", True, False),
+                       Range("3F", "7F", True, True),
+                       False),
+                      (Range("3F", "7F", True, False),
```

Comment:
We can add a scenario for _point_ range.

Reply:
added point range tests